stable/mlrun/templates/db-configmap-init.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if eq .Values.httpDB.dbType "mysql" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mlrun.db.initConfigMapName" . }}
+  labels:
+    {{- include "mlrun.db.labels" . | nindent 4 }}
+data:
+  enable-root-remote-access.sql: |
```

Comment:
Just making sure that you copy pasted the changes from development and not your original PR that was targeted for mysql 5.6 and not 5.7